tools/topology/topology2/include/common/pcm_caps.conf

```diff
@@ -88,8 +88,8 @@ Class.PCM."pcm_caps" {
 	periods_max		16
 	channels_min		2
 	channels_max		2
-	period_size_min		192
-	period_size_max		16384
-	buffer_size_min		65536
-	buffer_size_max		65536
+	period_size_min		192 # Stereo, S16_LE, 48KHz, 1ms
+	period_size_max		19200 # Stereo, S16_LE, 48KHz, 100ms
+	buffer_size_min		384 # period_size_min * periods_min
+	buffer_size_max		307200 # period_size_max * periods_max
```

Comment:
I would make this size up to 10s, and also account for higher resolutions/number of channels. If we need to support 8ch 192kHz the buffers are going to be quite small in the end,

Reply:
For that we need to increase the periods_max.
These are the default parameters, if you override them then you need to update the sizes also.
In the definition we cannot use calculations, so it has to be done individually.

I can increase the periods_max to let's say 255?

Reply:
10s for all PCMs? You still need to select either the number of periods or the period size max.
These are the default PCM parameters, not the DB ones.
We can have UINT_MAX for both buffer and period size max and call it a day. 1024 max periods and all config should work.

But we likely want to specify _some_ numbers, right? 1ms min for capture, 2ms for playback, 100ms as max period size as default. Number of periods? 255 should be enough.
The you re-calculate it for places where the default is changed (format, channels, rate)
DeeBuffer is a special one, we need 100ms as minimum (the kernel needs to help here), but what max period size we want? 1s? 500ms? 10s max buffer is fine, I guess but I don't want to over complicate things.